Sodium/Sodium.swift

```diff
@@ -21,6 +21,7 @@ public class Sodium {
     public let keyExchange = KeyExchange()
     public let auth = Auth()
     public let stream = Stream()
+	public let keyDerivation = KeyDerivation()
```

Comment:
I think we ran into the great question of tabs vs spaces. I have my Xcode settings set to tabs, which I believe is the default and also recommended by the [GitHub Swift Style Guide](https://github.com/github/swift-style-guide/blob/master/README.md#whitespace). ;-)